utils/debug/3dTilesDebug.js

```diff
@@ -38,17 +37,13 @@ export default function create3dTilesDebugUI(datDebugTool, view, _3dTileslayer)
                     helper.position.copy(metadata.boundingVolume.region.position);
                     helper.rotation.copy(metadata.boundingVolume.region.rotation);
                     regionBoundingBoxParent.add(helper);
-                }
                 // 3dtiles with box
-                if (metadata.boundingVolume.box) {
-                    metadata.boundingVolume.box.getSize(size);
-                    const g = new THREE.BoxGeometry(size.x, size.y, size.z);
-                    const material = new THREE.MeshBasicMaterial({ wireframe: true });
-                    helper = new THREE.Mesh(g, material);
-                    metadata.boundingVolume.box.getCenter(helper.position);
-                }
+                } else if (metadata.boundingVolume.box) {
+                    bboxMesh.geometry.boundingBox = metadata.boundingVolume.box;
+                    helper = new THREE.BoxHelper(bboxMesh);
```

Comment:
Note: this could be used in `OBBHelper`.

Reply:
yes